packages/cli/src/commands/teams/invite.js

```diff
@@ -104,14 +90,14 @@ export default async function ({
     return 1;
   }
 
-  console.log(
-    info(introMsg || `Inviting team members to ${chalk.bold(currentTeam.name)}`)
+  output.log(
+    introMsg || `Inviting team members to ${chalk.bold(currentTeam.name)}`
   );
 
-  if (args.length > 0) {
-    for (const email of args) {
+  if (argv._.length > 0) {
```

Comment:
This length is always zero now, right? Since its hardcoded by the caller?

Reply:
Oh nevermind, I see there are two callers

Reply:
The hard-coding is when you do `vc teams add` (which finishes by allowing you to invite team members). But you can still do `vc teams invite xxx@example.com` in which case the length would not be zero.